test/bdd/pkg/waci/waci_issuance_sdk_steps.go

```diff
@@ -699,6 +674,275 @@ type prop interface {
+	if err != nil {
+		return nil, err
+	}
+
+	var presentationSubmission presexch.PresentationSubmission
+
+	err = json.Unmarshal(presentationSubmissionPRC, &presentationSubmission)
+	if err != nil {
+		return nil, err
+	}
+
+	documentLoader, err := createDocumentLoader()
+	if err != nil {
+		return nil, err
+	}
+
+	verifiableCredential, err := verifiable.ParseCredential(vcPRC,
+		verifiable.WithJSONLDDocumentLoader(documentLoader), verifiable.WithDisabledProofCheck())
+	if err != nil {
+		return nil, err
+	}
+
+	verifiableCredentials := []*verifiable.Credential{verifiableCredential}
+
+	proof := generateCredentialApplicationProof()
+
+	attachmentData := map[string]interface{}{
+		"@context":                cxt,
+		"type":                    types,
+		"credential_application":  credentialApplication,
+		"presentation_submission": presentationSubmission,
+		"verifiableCredential":    verifiableCredentials,
+		"proof":                   proof,
+	}
+
+	credentialApplicationAttachment := decorator.GenericAttachment{
+		ID:        uuid.New().String(),
+		MediaType: "application/json",
+		Format:    cm.CredentialApplicationAttachmentFormat,
+		Data:      decorator.AttachmentData{JSON: attachmentData},
+	}
+
+	return &credentialApplicationAttachment, nil
+}
+
+func generateCredentialManifest() (*cm.CredentialManifest, error) {
+	var credentialManifest cm.CredentialManifest
+
+	err := json.Unmarshal(credentialManifestDriversLicense, &credentialManifest)
+	if err != nil {
+		return nil, err
+	}
+
+	return &credentialManifest, nil
+}
+
+func generateCredentialFulfillmentAttachmentWithoutProof() (*decorator.GenericAttachment, error) {
+	var credentialFulfillment cm.CredentialFulfillment
+
+	err := json.Unmarshal(credentialFulfillmentDriversLicense, &credentialFulfillment)
+	if err != nil {
+		return nil, err
+	}
+
+	cxt := []string{
+		"https://www.w3.org/2018/credentials/v1",
+		cm.CredentialFulfillmentPresentationContext,
+	}
+
+	types := []string{
+		"VerifiablePresentation",
+		"CredentialFulfillment",
+	}
+
+	documentLoader, err := createDocumentLoader()
+	if err != nil {
+		return nil, err
+	}
+
+	vcPreview, err := verifiable.ParseCredential(vcDriversLicenseWithoutProof,
+		verifiable.WithJSONLDDocumentLoader(documentLoader))
+	if err != nil {
+		return nil, err
+	}
+
+	verifiableCredentials := []*verifiable.Credential{vcPreview}
+
+	attachmentData := map[string]interface{}{
+		"@context":               cxt,
+		"type":                   types,
+		"credential_fulfillment": credentialFulfillment,
+		"verifiableCredential":   verifiableCredentials,
+	}
+
+	credentialFulfillmentAttachment := decorator.GenericAttachment{
+		ID:        uuid.New().String(),
+		MediaType: "application/json",
+		Format:    cm.CredentialFulfillmentAttachmentFormat,
+		Data:      decorator.AttachmentData{JSON: attachmentData},
+	}
+
+	return &credentialFulfillmentAttachment, nil
+}
+
+func generateCredentialFulfillmentProof() map[string]string {
```

Comment:
Not sure if this is the best way to do this... let me know if there's something else I should do instead. Also let me know if `did:orb` is ok here.